LINK/etc/default/evm

```diff
@@ -25,6 +25,9 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 # Location of certificates and provider keys
 export KEY_ROOT=/var/www/miq/vmdb/certs
 
+export APPLIANCE_REPO_DIRECTORY=/opt/manageiq/manageiq-appliance
```

Comment:
Not sure I like the work repo here.  It implies it's cloned, which in some cases it may not be (like a rpm package).  Perhaps APPLIANCE_ROOT_DIRECTORY (or just _DIR since I'm lazy?)